src/phases/execution/MultiDeletes.yml

```diff
@@ -10,10 +10,26 @@ Description: >
 GlobalDefaults:
```

Comment:
Could you add a description block to this file, just like with the workloads? Keywords also. It's a nice way to organize the top level comments that we can programmatically use in the future.

Reply:
The file already had a description. Maybe it was hidden due to only showing the diffs?

Reply:
Yes, that's exactly what happened. Stupid github.